src/pykeen/trackers.py

```diff
@@ -85,3 +86,44 @@ def log_params(self, params: Dict[str, Any], prefix: Optional[str] = None) -> No
 
     def end_run(self) -> None:  # noqa: D102
         self.mlflow.end_run()
+
+
+class WANDBResultTracker(ResultTracker):
+    """A tracker for Weights & Biases. Note that you have to perform wandb login beforehand."""
+
+    def __init__(
+            self,
```

Comment:
please fix indentation (only 4 spaces more than the def block)

Reply:
Done in 0894bc36d0621671250f0f6b688e49e4e1bea6ed.